### ISSUE TITLE
Creating a new recipe fails with error 500

### ISSUE THREAD

**Issue by Forceu** [vabene1111/recipes#718]
### Version
Version: 0.16.6

### Bug description
When trying to create a new recipe, it fails with error 500. This happens after entering the name.

```
ERROR:django.request:Internal Server Error: /new/recipe/
Traceback (most recent call last):
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "space_id" violates not-null constraint
DETAIL:  Failing row contains (120, TEXT, , 0, , 0, t, null, null).


The above exception was the direct cause of the following exception:

ERROR:django.request:Internal Server Error: /new/recipe/
Traceback (most recent call last):
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "space_id" violates not-null constraint
DETAIL:  Failing row contains (120, TEXT, , 0, , 0, t, null, null).


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/recipes/cookbook/helper/permission_helper.py", line 153, in dispatch
    return super(GroupRequiredMixin, self).dispatch(request, *args, **kwargs)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/views/generic/edit.py", line 172, in post
    return super().post(request, *args, **kwargs)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/views/generic/edit.py", line 142, in post
    return self.form_valid(form)
  File "/opt/recipes/cookbook/views/new.py", line 44, in form_valid
    obj.steps.add(Step.objects.create())
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/models/query.py", line 453, in create
    obj.save(force_insert=True, using=self.db)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/models/base.py", line 726, in save
    self.save_base(using=using, force_insert=force_insert,
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/models/base.py", line 763, in save_base
    updated = self._save_table(
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/models/base.py", line 868, in _save_table
    results = self._do_insert(cls._base_manager, using, fields, returning_fields, raw)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django_prometheus/models.py", line 43, in _do_insert
    return super()._do_insert(*args, **kwargs)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/models/base.py", line 906, in _do_insert
    return manager._insert(
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/models/query.py", line 1270, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1416, in execute_sql
    cursor.execute(sql, params)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 66, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 75, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/opt/recipes/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.IntegrityError: null value in column "space_id" violates not-null constraint
DETAIL:  Failing row contains (120, TEXT, , 0, , 0, t, null, null).
```

**Comment by vabene1111** [comment]
Are you running latest or a version tag 0.16.6? I have pushed a hot fix yesterday but hadn't yet had the time to deploy it as a taggers version?

**Comment by Forceu** [comment]
I just tried it with the latest `develop` docker image and it happens there as well, even on a new instance.

**Comment by vabene1111** [comment]
develop is actually behind master at the moment as i am working in the browser from my mobile phone and did not yet have the time to merge the branches :joy: just 

0.16.7 is currently publishing and `latest` should also work, trying to merge into develop as well

**Comment by sqozz** [comment]
hm, I see a similar problem even on master/0.16.7 (tried both). However I don't see a stack trace. All I can observe is a warning from django:

```
/opt/tandoor/venv/lib/python3.9/site-packages/django/db/models/fields/__init__.py:1416: RuntimeWarning: DateTimeField Recipe.created_at received a naive datetime (2021-06-13 17:07:49.775101) while time zone support is active.
  warnings.warn("DateTimeField %s received a naive datetime (%s)"
```

worth a new bug?

**Comment by Forceu** [comment]
@vabene1111 Seems to be fixed in 1.16.7, you I close this issue :)
@sqozz It's probably a good idea to open a new issue for that
